internal/pkg/util/configurebccsp.go

```diff
@@ -59,8 +60,15 @@ func ConfigureBCCSP(optsPtr **factory.FactoryOpts, mspDir, homeDir string) error
 		log.Debugf("Initializing BCCSP with software options %+v", opts.SwOpts)
 	}
 	if opts.Pkcs11Opts != nil {
-		log.Debugf("Initializing BCCSP with PKCS11 options %+v", opts.Pkcs11Opts)
+		log.Debugf("Initializing BCCSP with PKCS11 options %+v", sanitizePkcs11Opts(*opts.Pkcs11Opts))
 	}
 	*optsPtr = opts
 	return nil
 }
+
+// redacts label and pin from PKCS11 opts
+func sanitizePkcs11Opts(opts pkcs11.PKCS11Opts) pkcs11.PKCS11Opts {
```

Comment:
can you add a quick test to make sure this works?

Reply:
It's an unexported function, only used when logging at debug level.

Reply:
You could still technically test the logger output or just have the test file in the same package as util. I guess I'm indifferent either way, will leave up to the ca maintainers to approve and merge if they feel a test is unneeded.

Reply:
Sure, I added a new test file specifically for pkcs11 configuration.

Reply:
If we're doing this to fix a bug, there should be a test. If there's no test, there's no reason to keep the behavior.

On a related note, I'm pretty sure the pkcs11 tag isn't even set when running unit tests so that should make this even more fun.